priv/schema/rabbit.schema

```diff
@@ -1068,6 +1068,14 @@ end}.
 {mapping, "vhost_restart_strategy", "rabbit.vhost_restart_strategy",
     [{datatype, {enum, [stop_node, continue, transient, persistent]}}]}.
 
+%% Approximate maximum time a consumer can spend processing a message before
+%% the channel is terminated, in milliseconds.
+%% {consumer_timeout, 10000},
+
+{mapping, "consumer_timeout", "rabbit.consumer_timeout", [
+    {datatype, integer}
```

Comment:
Should we add a "positive integer" validator?

Reply:
If that is an option then yeah.

Reply:
OK I'll get that too